scripts/go_fmt.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then
+    gofmt -d ./...
```

Comment:
Because there's the `-s` flag (`simplify`) in the `if` statement, we should also add it here to have a consistent output.

Otherwise, we would exit with code `1` because something could be simplified, but the `gofmt -d` would not report anything. I feel like we'd need `gofmt -d -s` here.

Reply:
I've tested this in a personal project with a PR:
https://github.com/tpotjj/simplebank/actions/runs/3180595633/jobs/5184359497

The previous "." looks sufficient there, also for showing multiple formatting 'errors' in different files/subdirs.

Or, do I miss something?

Reply:
:astonished: did not know that! I looked it up now: https://stackoverflow.com/a/13333931/3243814

You are absolutely correct. The difference between `gofmt` and `go fmt` :sweat_smile:

Reply:
@tpotjj The `-s` would still be needed in both, though.

Reply:
Resolved this suggestion:
Small mistake between the usage of `go fmt` && `gofmt`